atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/TableMigratorTest.java

```diff
@@ -139,7 +152,13 @@ public void testMigrationToDifferentKvs() {
                 DefaultTimestampCache.createForTests(),
                 MultiTableSweepQueueWriter.NO_OP,
                 TransactionKnowledgeComponents.createForTests(kvs2, metricsManager.getTaggedRegistry()),
-                MoreExecutors.newDirectExecutorService());
+                MoreExecutors.newDirectExecutorService(),
+                new DefaultKeyValueSnapshotReaderManager(
+                        new DelegatingTransactionKeyValueServiceManager(kvs2),
+                        transactionService,
+                        false,
+                        new DefaultOrphanedSentinelDeleter(ssm2::get, verifyDeleteExecutor),
+                        verifyDeleteExecutor));
```

Comment:
non-blocking: yeah don't know if there's a good way to reduce duplicaiton on creating DKVSRManagers from a few primitives across the tests, I imagine we repeat this block a lot, though can buy that there are enough variants that writing a generic version might be hard